data/desktop.ini

```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
+ECLIPSE_base_CLE_V5a_NOx.nc=@ECLIPSE_base_CLE_V5a_NOx.nc,0
```

Comment:
What is the purpose of this file?

Reply:
I have no idea what that is... I just noticed that, it was created automatically for reasons I ignore..

Reply:
removed